BurraAbhishek/SpeechVision
# Update app icon

What is the purpose of this pull request? #
Resolves #1 

Describe the proposed changes: #
Replace the current app icons with the icon described in #1 

How to test those changes? #
Run the application. If you're using Android Studio:
1. Run the application
2. Close the application on your emulator/physical device
The new icon should now be visible.
